packages/k/kmscon/package.yml

```diff
@@ -0,0 +1,34 @@
+# yaml-language-server: $schema=/usr/share/ypkg/schema/schema.json
+name       : kmscon
+version    : 9.2.1
+release    : 1
+source     :
+    - https://github.com/kmscon/kmscon/archive/refs/tags/v9.2.1.tar.gz : d801a3f0bbfff6940eb249ade41c28307ef846132b4e10a499abe76d535e24e8
+homepage   : https://www.freedesktop.org/wiki/Software/kmscon
+license    :
+    - MIT
+    - LGPL-2.1-or-later # external/htable.* external/dbus-loop.*
+component  : system.utils
+summary    : Linux KMS/DRM based virtual Console Emulator
+description: |
+    kmscon is a system console for linux. It does not depend on any graphics-server on your system (like X.org), but instead provides a raw console layer that can be used independently. It can replace the linux kernel console entirely but was designed to work well side-by-side, too. Even though initially targeted at providing internationalization to the system-console, it has grown into a fully modularized console layer including features like multi-head support, internationalized font rendering, XKB-compatible keyboard handling, hardware-accelerated graphics access and more.
+builddeps  :
+    - pkgconfig(libtsm)
+    - pkgconfig(libudev)
+    - pkgconfig(xkbcommon)
+checkdeps  :
+    - pkgconfig(check)
+setup      : |
+    %meson_configure
+build      : |
+    %ninja_build
+install    : |
+    %ninja_install
```

Comment:
Could I ask you to install the project's license file with `%install_license COPYING`?